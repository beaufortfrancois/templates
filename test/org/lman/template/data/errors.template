{{?_.errors}}
  There are no errors yet. This shouldn't happen.
{{/_.errors}}
{{_.errors}}
{{foo}} will generate an error.
{{bar}} will not.
{{_.errors}}
{{baz}} will generate another error.
{{_.errors}}
