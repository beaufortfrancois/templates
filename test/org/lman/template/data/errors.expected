 will generate an error.
bar will not.
Failed to resolve '_.errors' at line 4 column 3 in &lt;root&gt;
Failed to resolve 'foo' at line 5 column 3 in &lt;root&gt;
 will generate another error.
Failed to resolve '_.errors' at line 4 column 3 in &lt;root&gt;
Failed to resolve 'foo' at line 5 column 3 in &lt;root&gt;
Failed to resolve 'baz' at line 8 column 3 in &lt;root&gt;
